go-controller/vendor/github.com/coreos/go-iptables/iptables/iptables.go

```diff
@@ -542,6 +682,15 @@ func getIptablesCommand(proto Protocol) string {
 	}
 }
 
+// getIptablesRestoreCommand returns the correct command for the given protocol, either "iptables" or "ip6tables".
```

Comment:
nit doc string again :D

Reply:
ok this ones mine :) i'll defer fixing it unless there is something else that needs to be fixed in this vendored code